extensions/applicationinsights-analytics-js/package.json

```diff
@@ -33,7 +33,10 @@
     },
     "dependencies": {
         "@microsoft/applicationinsights-core-js": "2.4.0-beta",
-        "@microsoft/applicationinsights-common": "2.4.0-beta",
+        "@microsoft/applicationinsights-common": "2.4.0-beta"
+    },
+    "peerDependencies": {
+        "@microsoft/applicationinsights-rollup-es3" : "1.0.0",
```

Comment:
See other comments. I think this can be a dev dep

Reply:
Moved as suggested